rls/src/main/java/io/grpc/rls/RlsRequestFactory.java

```diff
@@ -73,12 +73,12 @@ RouteLookupRequest create(String service, String method, Metadata metadata) {
       grpcKeyBuilder = keyBuilderTable.get("/" + service + "/*");
     }
     if (grpcKeyBuilder == null) {
-      return new RouteLookupRequest(ImmutableMap.<String, String>of());
+      return RouteLookupRequest.create(ImmutableMap.<String, String>of());
```

Comment:
The type seems to be inferable now 
```suggestion
      return RouteLookupRequest.create(ImmutableMap.of());
```

Reply:
This is due to the recent drop of Java 7 support. We will do a systematic clean up to java8-tify the codebase after a release.

Reply:
Got it, didn't know.